internal/oci/container.go

```diff
@@ -755,3 +756,18 @@ func (c *Container) SetResources(s *specs.Spec) {
 func (c *Container) GetResources() *types.ContainerResources {
 	return c.resources
 }
+
+// SetPlatformForImage sets the platform of the image for container.
+func (c *Container) SetPlatformForImage(os, arch string) { //nolint:gocritic
```

Comment:
what was gocritic complaining about?

Reply:
It's about `os`, since we are also importing `os` package, it got confused with that keyword here

Reply:
can you use `OS` instead so it doesn't shadow?